### DISCUSSION TITLE
Extension - Zoom for tabbed component.

### DISCUSSION THREAD

**Discussion by Symbiomatrix** [richrobber2/canvas-zoom#56] [Q&A] [answered]
Hello again, @daswer123.
I've added a [branch](https://github.com/Symbiomatrix/sd-webui-regional-prompter/tree/Tabs1) for regional which puts the various modes & respective components, including polymask, under tabs. I noticed preset loading breaks the component completely if it's run before entering the mask tab for the first time, so I wondered whether it causes issues for zoom too. I'm still on the fence with this version. Would be great to hear your opinion.

**Comment by daswer123** [comment, marked as answer]
Hi, I checked and canvas-zoom works fine, nothing breaks. 
And in general I think tabs are a good solution for more compatibility of the extension

**Reply by Symbiomatrix** [reply]
Thanks, chum.
